src/main/java/pl/psnc/dei/service/TranscriptionPlatformService.java

```diff
@@ -44,7 +41,7 @@
 @Transactional
 public class TranscriptionPlatformService {
 
-	private static final int READ_TIMEOUT_IN_SECONDS = 5;
+	public static final int READ_TIMEOUT_IN_SECONDS = 600;
 	private static final int WRITE_TIMEOUT_IN_SECONDS = 5;
```

Comment:
Why is this const public? Is that really necessary?

Reply:
to use it in timeout in tests that makes you don't have to change tests after timeout changes